latexify/predefined_variables.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ast
+
+def get_math_symbols():
+  math_symbols = {
```

Comment:
Placing this and following dictionaries onto the module looks better rather than wrapping getter functions. Module level constants will be initialized only once and can be accessed directly from others.

Reply:
Thank you for the feedback.
I would like to get some clarification - I should initialize the constants at the global scope of `core.py` instead of calling the getter functions at function-level to avoid being initialized multiple times? Do I understand your intention correctly?

e.g. at `core.py
```
MATH_SYMBOLS = constants.get_math_simbols()

def visit_xxx():
    some code
```

Reply:
I prefer module-level constants
```python
# constants.py
MATH_SYMBOLS = {
  ...
}
```
encouraged at https://github.com/google/styleguide/blob/gh-pages/pyguide.md#254-decision

Reply:
@ShigekiKarita Thank you! I guess that is what @odashi actually meant. I will go in that approach.

Reply:
Fixed in the following commits.